tests/gflownet/envs/test_grid.py

```diff
@@ -60,3 +72,7 @@ def test__statebatch2oracle__returns_expected(env, states, statebatch2oracle):
 
 def test__all_env_common(env):
     return common.test__all_env_common(env)
+
+
+def test__all_env_common(env_extended_action_space):
```

Comment:
Common tests are great, but I would suggest also having some manual ones for `step`/`get_action_space`/`get_mask_invalid_actions_forward` functions where there is a couple of edge cases and expected output (for instance, for the bug in the previous version where invalid actions were not caught if there were multiple increments of the same dimension).